src/main/resources/com/adobe/epubcheck/messages/MessageBundle.properties

```diff
@@ -293,6 +293,7 @@ RSC_019=EPUBs with Multiple Renditions should contain a META-INF/metadata.xml fi
 RSC_020='%1$s' is not a valid URI.
 RSC_021=A Search Key Map Document must point to Content Documents ('%1$s' was not found in the spine).
 RSC_022=Cannot check image details (requires Java version 7 or higher).
+RSC_023='%1$s' is not a valid URL.
```

Comment:
Yeah, you're right, that's technically wrong. I was trying to phrase this more user friendly.

`URL '%1$s' has no host component` is technically correct, but probably hard to understand for non-geeks... ;-)

If we phrase it more technical I would like to add something like `…component. Probably misspelled or missing http(s):// ?`. But I'm not sure whether this makes for a good warning message...